samples/account-linking/csharp/Source/Services/OAuth/OAuthServiceClient.cs

```diff
@@ -75,8 +75,11 @@ private HttpRequestMessage CreateCodeClaimRequest(string code)
             {
                 { new StringContent(_oauthClientId), "client_id" },
                 { new StringContent(_oauthClientSecret), "client_secret" },
-                { new StringContent(code), "code" }
-            },
+                { new StringContent(code), "code" },
+                { new StringContent("authorization_code"),"grant_type" },
```

Comment:
Same here, please add comment explaining why this is needed for Gmail.

Reply:
I wouldn't this is actually spec OAuth code flow. It was just an idiosyncrasy that the GitHub Auth server implicitly assumed the grant type.

Reply:
Included reference link here - https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/320/commits/7b026e28f045fcd72175c14b59fa0f874602baef